aip/general/0121.md

```diff
@@ -94,6 +94,33 @@ the standard methods. Custom methods offer the same design freedom as
 traditional RPC APIs, which can be used to implement common programming
 patterns, such as database transactions, import and export, or data analysis.
 
+### Consistency
+
+For methods that operate on a resource, the completion of those operations
+(either successful or with an error) **must** mean that the state of the
+resource's existence and all user-settable values have reached a steady-state.
+
+[output only][] values **should** also have reached a steady-state, except
+for values that are related to the resource [state][].
+
+Examples include:
+
+- A get request for a resource immediately after the successful completion of a
+  create method **must** return the resource.
+- A get request for a resource immediately following an update **must** return
+  a resource with the final values from the update request.
```

Comment:
What about delete?

Reply:
ack. These were just examples, but I suppose it doesn't hurt to add a delete.